### DISCUSSION TITLE
Uncaught error: Sanity LocaleContext value missing. Is this hook being used outside LocaleContext.Provider?

### DISCUSSION THREAD

**Discussion by ZeenatFirdosh** [sanity-io/sanity#6142] [Sanity Studio Support]
While inserting data and after hitting publish, I am getting this error.
![image](https://github.com/sanity-io/sanity/assets/100707152/a7c5f904-4ae8-4dde-b62b-192f9f5c9216)
os: windows 10

**Comment by oddjob23** [comment]
Give this a try: https://www.sanity.io/changelog/2f1d69c0-1d5f-4bd7-9607-fa1b27d8648d

I had the same error, with the upgrade the issue is resolved. Hope it helps

**Comment by Coderamrin** [comment]
Upgrading to the latest version fixed the issue for me too.
